jwst/photom/photom.py

```diff
@@ -1099,7 +1195,8 @@ def apply_photom(self, photom_fname, area_fname):
 
         # Load the pixel area reference file, if it exists, and attach the
         # reference data to the science model
-        self.save_area_info(ftab, area_fname)
+        if self.exptype != 'NIS_SOSS':
```

Comment:
Might want to add a comment indicating why this isn't applicable to SOSS (for future maintainers)